fr/releases/centreon-commercial-extensions.md

```diff
@@ -152,6 +152,16 @@ liens
 
 ## Centreon BAM
 
+### 20.04.9
+
+`January 07, 2022`
+
+#### Bugfixes
+
+- [Configuration] Fixed an error when generating config if a disabled BA is connected to an escalation task
```

Comment:
```suggestion
- [Configuration] Fixed an error when generating config if a disabled BA is connected to an escalation task
```

Reply:
No difference

Reply:
You probably meant to copy this:
[Configuration] Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
```suggestion
[Configuration] Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
```